js/countryToLanguageMapping.js

```diff
@@ -55,7 +55,7 @@ var countryToLanguageMapping = {
     'cx': 'en',
     'cy': 'el,tr',
     'cz': 'cs',
-    '// de': 'de',
+    // 'de': 'de',
```

Comment:
Ups, thanks!

Reply:
You're welcome! But I still don't understand why some lines are commented out 😅

Reply:
Check the function in the same file. I want to rely on the fallback behavior of the function. Mapping de to de provides no information. I guess the commented out lines can be removed.